src/modules/common/components/RemoteContent.js

```diff
@@ -10,12 +10,13 @@ class RemoteContent extends React.Component {
       PropTypes.shape({
         __html: PropTypes.string.isRequired
       }).isRequired
-    ).isRequired
+    ).isRequired,
+    isCentered: PropTypes.bool
```

Comment:
I think a more common name would be just `centered` instead of `isCentered` here.

Reply:
done